pkg/utils/helper.go

```diff
@@ -11,6 +12,59 @@ import (
 	"strings"
 )
 
+type Azure struct {
+	Cloud string `json:"cloud"`
+}
+
+type AzureStackCloud struct {
+	StorageEndpointSuffix string `json:"storageEndpointSuffix"`
+}
+
+// IsAzureStackCloud returns true if the application is running on Azure Stack Cloud
+func IsAzureStackCloud() bool {
+	azureFile, err := RunCommandOnHost("cat", "/etc/kubernetes/azure.json")
+	if err != nil {
+		return false
+	}
+	var azure Azure
+	err = json.Unmarshal([]byte(azureFile), &azure)
+	if err != nil {
+		return false
+	}
+	cloud := azure.Cloud
+	return strings.ToUpper(cloud) == "AZURESTACKCLOUD"
+}
+
+// CopyCert saves the cert content to a new file inside the container
+func CopyCert() error {
+	cert, err := RunCommandOnHost("cat", "/etc/ssl/certs/azsCertificate.pem")
+	if err != nil {
+		return fmt.Errorf("Failed to retrieve cert content: %+v", err)
+	}
+	err = WriteToFile("/etc/ssl/certs/azsCertificate.pem", cert)
+	if err != nil {
+		return fmt.Errorf("Failed to write to file: %+v", err)
+	}
+	return nil
+}
+
+// GetStorageEndpointSuffix returns the SES url from the JSON file as a string
+func GetStorageEndpointSuffix() string {
+	if IsAzureStackCloud() {
+		ascFile, err := RunCommandOnHost("cat", "/etc/kubernetes/azurestackcloud.json")
+		if err != nil {
+			return "core.windows.net"

```

Comment:
define a constant variable to store "core.windows.net" and consume the constant variable